### DISCUSSION TITLE
Session recording based on labels

### DISCUSSION THREAD

**Discussion by mrik974** [gravitational/teleport#31868] [Q&A] [answered]
Hi,
I have a bunch of sensitive servers (the backup servers for instance) that are sensitive. but I also have 10ths of other servers that are not so sensitive.
Is there a way to configure Teleport to only record sessions based on the labels I put on the sensitive servers ?

**Comment by stevenGravy** [comment, marked as answer]
Hi @mrik974 , no that is not currently an option.  You may want to run those servers through another cluster with no recording and use the trusted cluster option.

**Comment by mrik974** [comment]
Thanks for the answer, and for the idea. I could also clean the sessions I don’t want to keep. Is there a good practice for this ?

From: Steven Martin ***@***.***>
Date: Friday, 15 September 2023 at 14:35
To: gravitational/teleport ***@***.***>
Cc: mrik974 ***@***.***>, Mention ***@***.***>
Subject: Re: [gravitational/teleport] Session recording based on labels (Discussion #31868)

Hi @mrik974<https://github.com/mrik974> , no that is not currently an option. You may want to run those servers through another cluster with no recording and use the trusted cluster option.
